app/src/main/java/chat/rocket/android/chatroom/ui/MessageFlexLayout.kt

```diff
@@ -0,0 +1,126 @@
+package chat.rocket.android.chatroom.ui
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.util.AttributeSet
+import android.view.View
+import android.widget.RelativeLayout
+import android.widget.TextView
+import chat.rocket.android.R
+
+class MessageFlexLayout : RelativeLayout {
+    private var viewMain: TextView? = null
+    private var viewSlave: View? = null
```

Comment:
Though it used to be very common usage, people are generally trying to avoid using "slave" more recently. Especially in cases where it hasn't already been introduced. Would "Primary/Secondary" make sense here as a replacement for "Main/Slave"?